README.md

```diff
@@ -1,2 +1,2 @@
 # indonesia-holiday
-🇮🇩 Check Indonesian holiday time
+🇮🇩 Check Indonesian holiday time
```

Comment:
I am a noob. Would you mind to add steps on how to set up, dependencies, etc :baby:

Reply:
at the first time this is just init, so there will be step to add `readme.md`, 😁